apache/superset
# chore: Bump plugin-chart-pivot-table to 0.17.57

### SUMMARY
Bump `@superset-ui/plugin-chart-pivot-table` to `^0.17.57` in order to integrate the changes from https://github.com/apache-superset/superset-ui/pull/1172.

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF
See video in https://github.com/apache-superset/superset-ui/pull/1172

### TESTING INSTRUCTIONS
1. Create a Pivot Table v2 chart
2. Use "Apply metrics on" button to change metrics's layout and verify that the results are correct

### ADDITIONAL INFORMATION



CC: @villebro @rusackas
